ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -194,4 +217,25 @@ open class ContactView: UIView {
 
         subtitleLabel = label
     }
+
+    private func setupPressedStateOverlay() {
+        pressedStateOverlay.clipsToBounds = true
+        pressedStateOverlay.frame = avatarView.frame
+        pressedStateOverlay.layer.cornerRadius = avatarView.frame.width / 2
+    }
+
+    open override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
+        super.touchesBegan(touches, with: event)
+        pressedStateOverlay.backgroundColor = UIColor(red: 0.83, green: 0.83, blue: 0.83, alpha: 0.6)
+    }
+
+    open override func touchesMoved(_ touches: Set<UITouch>, with event: UIEvent?) {
+        super.touchesMoved(touches, with: event)
+        pressedStateOverlay.backgroundColor = UIColor(red: 1.0, green: 1.0, blue: 1.0, alpha: 0.0)
```

Comment:
Shouldn't we set the "pressed" color back if the touches are within the bounds of the view?

Reply:
Can you elaborate more on what you mean by this? From what I understand, you're saying that we should still be setting the color to the "pressed" color if the touches has moved to somewhere still within the `ContactView`? Please correct me if this isn't your intention.

Reply:
Take a look at the button control in the demo app.
If you do a touch down, touch move (in and out of the control bounds), touch up, depending on where your at, the control will be dimmed or not dimmed. ContactView needs to have the same behavior as Buttons for the touch down/move/up behavior.

Reply:
Do we want the same behaviour as the Buttons in the demo app? I check the iOS Outlook app and it seems that when we click and move, the grey overlay disappears as well. Which behaviour should we adhere to?

Reply:
Let's go with what Outlook iOS does.